k8s/README.md

```diff
@@ -0,0 +1,13 @@
+Kubernetes Secrets Provider
+
+To run the test export the following environment variable
+
+```
+$ export KUBECONFIG=/path/to/kubeconfig/file
+```
```

Comment:
add the command to create the secret needed by the test

Reply:
The test itself creates the secret.